embark.el

```diff
@@ -2804,6 +2804,24 @@ before or after the sexp (those are the two locations at which
 (embark--sexp-command raise-sexp)
 (embark--sexp-command mark-sexp)
 
+(defun embark-next-symbol (sym)
+  "Jump to next SYM without an error in case the symbol is not found.
```

Comment:
I think I might completely rewrite these:

1. The docstring is written with the point of view of justifying the existence of the command, not explaining what it does! It reads very defensive.
2. I'd prefer using the no-error argument than condition-case.
3. If we did use condition-case, we should probably just catch `search-failed`. Here it's probably no problem to be broad, but I like catching specific errors to avoid hiding others.

Reply:
Oh, also, the interactive prompt could be nicer.

Reply:
> The docstring is written with the point of view of justifying the existence of the command, not explaining what it does! It reads very defensive.

I am defensive. From what I've heard from you, you don't like this functionality. I don't want to push functionality in your library which you seem to dislike. But maybe I sensed this wrongly. Just in case, there are more nice use cases, string-inflection, evil-numbers-like functionality, adjusting color values at point etc. So I like this and would like to see if turns out to replace many hydra use cases, since I very much prefer Embark over Hydra since its keymap usage is more lightweight. For non-at-point commands I am playing around with repeat-mode.

> I'd prefer using the no-error argument than condition-case.

Oh you are very right about that.

> If we did use condition-case, we should probably just catch search-failed. Here it's probably no problem to be broad, but I like catching specific errors to avoid hiding others.

Also right. But we should really use noerror.

Reply:
I don't dislike the repeatable action functionality, I just don't think it's very necessary. But it's in now and I fully support it. I just don't think the debate about it should go into docstrings, which should just matter-of-factly describe the command. The evil numbers example is nice, I might even want that in my config!

I'll push a commit in a few minutes rewriting these.

Reply:
Pushed the update with the noerror argument.